### DISCUSSION TITLE
DLC not dumping

### DISCUSSION THREAD

**Discussion by Darwock** [hippie68/ftpdump#26] [Q&A] [answered]
Hi, thanks for this tool - I was very hopeful it would solve my problems but I’m hitting a snag dumping DLC.

When I do the —list option I can see 5/5 DLC items against the game ID I am trying to dump. However when I run the game, and dump with just the —dlc option it tells me no DLC is found for the game ID. Same thing if I do a full dump, I get the app data and patch data then a message that no DLC was found.

My only glimmer of hope is because the list option *does* pick up on the 5 dlc items that are installed. It’s just the dump process that can’t find them somehow.

Any advice?

**Comment by hippie68** [comment, marked as answer]
The script can only dump data that is permanently exposed by the PS4 (decrypted and mounted) while a game is running. I have not seen publicly shared methods to dump data that is only temporarily exposed (allegedly some unlocking DLC is loaded only for a split second). However, if the DLC simply unlocks something, be aware there are unlocker tools out there. I have not used such tools and I think there may be legal issues, so I won't link to them. If the DLC is not supposed to unlock something, then there may be a script problem (please clarify).

**Reply by Darwock** [reply]
Thanks for the reply. It’s DLC data for character costumes and as I understand it, the data is included in the DLC file and is not data already in the main app that is just opened with a DLC unlock key.

I guess it’s a quirk of this game in how it loads the DLC then? Frustratingly there are fpkg dumps out there of other equivalent DLC from the same game so there must be a way!

edit - I found some discussion on dumping the DLC and can confirm it is this game in particular that is troublesome, and not a problem with your script. Thanks for your time!
